Makefile

```diff
@@ -28,38 +28,43 @@ PUBLISH := $(shell ./scripts/cicd.sh publish?)
 
 #####################################################################
 
-.PHONY: checks deps gofmt \
+.PHONY: checks check-go check-docker deps gofmt \
 	ci build build-cross \
 	docker-build docker-push \
 	bin-build bin-push \
 	clean help
 
-checks:  ## Check the system before building
-	./scripts/cicd.sh checks
+checks: check-go check-docker ## Check the system before building
```

Comment:
^ should have two spaces before `##` to follow existing pattern.

Reply:
Done